src/Related/components/Card.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import GalleryCarousel from '../../Widgets/GalleryCarousel.jsx';
+
+const Card = ({details, images}) => {
```

Comment:
Unfamiliar with how destructuring would work in passing props, but looks interesting.